build.gradle

```diff
@@ -28,6 +28,14 @@ subprojects {
     if (project.name == "docker-init-script") {
         return
     }
+
+
+    if (project.path.startsWith(':legacy')) {
+        return
+    }
+
+    println "root build.gradle: configuring ${project.name}"
```

Comment:
Use `logger.info` or `logger.debug` instead of `println`

Reply:
Yes, there is a ticket for this: https://xenitsupport.jira.com/browse/DOCKER-256 (piling up)

Reply:
I'm just going to remove that line, it was for debugging purposes